distribution/docs/src/main/resources/content/_configuring/importing-exported-configurations-contents.adoc

```diff
@@ -8,18 +8,21 @@
 
 [WARNING]
 ====
-Importing of configurations must take place instead of performing installation for the first time and after performing OS-level hardening.
+Importing of configurations must take place after performing installation for the first time and after performing OS-level hardening.
```

Comment:
Should we also note that there will be some additional OS level hardening to do after migration:import (file perms, etc.)?